hare/hare.go

```diff
@@ -172,13 +172,18 @@ func (h *Hare) onTick(id types.LayerID) {
 		set.Add(Value{b})
 	}
 
-	instid := InstanceId(id)
-
-	cp := h.factory(h.config, instid, set, h.rolacle, h.sign, h.network, h.outputChan)
-	cp.SetInbox(h.broker.Register(cp.Id()))
+	instId := InstanceId(id)
+	c, err := h.broker.Register(instId)
+	if err != nil {
+		h.Warning("Could not register CP to broker err=%v", err)
```

Comment:
add the layer id to the log print, otherwise it will be very hard to debug issues around this

Reply:
done